atuin-server/src/database.rs

```diff
@@ -141,6 +149,46 @@ impl Database for Postgres {
         Ok(res.0 as i64)
     }
 
+    async fn delete_history(&self, user: &User, id: String) -> Result<()> {
+        sqlx::query(
+            "update history
+            set deleted_at = $3
+            where user_id = $1
+            and client_id = $2
+            and deleted_at is null", // don't just keep setting it
```

Comment:
Interesting. I assume this can avoid a write flush

Reply:
Probably, but also means deleted_at won't be set if it has already been set